src/lib/IONOS/components/IONOS_Footer.svelte

```diff
@@ -0,0 +1,51 @@
+<script lang="ts">
+	const currentYear = new Date().getFullYear();
+</script>
+
+<footer>
+	<div class="footer-right"></div>
+	<div id="copyright">
+		<p>© {currentYear} IONOS SE</p>
+	</div>
+	<div class="footer-left">
+		<a href="https://www.ionos.de/impressum" target="_blank">Impressum</a>
```

Comment:
* let's do it via i18n module in order to be able faster introduce second language/language specific URLs